koku/api/apps.py

```diff
@@ -14,8 +14,70 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
+"""API application configuration module"""
+
+import logging
+
 from django.apps import AppConfig
 
+from koku.env import ENVIRONMENT
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
 
 class ApiConfig(AppConfig):
+    """API application configuration"""
+
     name = 'api'
+
+    def ready(self):
+        """Action on application startup."""
+        # noqa: E402 pylint: disable=C0413
+        from django.db.utils import OperationalError
```

Comment:
Why not top level imports here (and in the below functions, too)?

Reply:
Importing before the ready function causes the application to blow up as all server capabilities haven't been loaded and made ready. I had actually dig into that. #stackOverflow